src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -83,6 +83,10 @@
             if (ret = allHeaders.match(/^x-debug-token-link:\s+(.*)$/im)) {
                 stackElement.profilerUrl = ret[1];
             }
+            if (ret = allHeaders.match(/^x-debug-toolbar-replace:\s+(.*)$/im)) {
```

Comment:
what's the suggested value here? `"0" / "1"`?

Reply:
See line 90 below as well as the updated change log. It will accept any truthy value.